# fix: /«PKGBUILDDIR»/src/chatty/gui/components/settings/SettingsDialog.java:155: error: unmappable character for encoding ASCII
Index: chatty-0.6/src/chatty/gui/components/settings/SettingsDialog.java
===================================================================
--- chatty-0.6.orig/src/chatty/gui/components/settings/SettingsDialog.java	2014-02-13 13:27:11.000000000 +0100
+++ chatty-0.6/src/chatty/gui/components/settings/SettingsDialog.java	2014-02-14 18:13:00.071685942 +0100
@@ -152,7 +152,7 @@
         gbc = makeGbc(0,2,1,1);
         gbc.anchor = GridBagConstraints.NORTHWEST;
         gbc.insets = new Insets(0,10,0,0);
-        add(new LinkLabel("[mäh:muh Help]", new LinkLabelListener() {
+        add(new LinkLabel("[mah:muh Help]", new LinkLabelListener() {
 
             @Override
             public void linkClicked(String type, String ref) {
